packages/documentation/gatsby-browser.js

```diff
@@ -0,0 +1,14 @@
+exports.onInitialClientRender = (n, config) => {
+  // eslint-disable-next-line
+  window.mermaid_config = { theme: config.theme || 'default', startOnLoad: true }
+  // load initial script
+  const s = document.createElement('script');
+  s.setAttribute('src', 'https://unpkg.com/mermaid@7.1.0/dist/mermaid.min.js');
+  document.head.appendChild(s);
+  // XXX: ugly hack because onRouteUpdate doesn't know when the react is done
+  setInterval(() => {
```

Comment:
We might be able to use some lifecycle methods in [this](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8efd9de68d277c1218eba52d289839c7e5786c1d/packages/documentation/src/layouts/module-components.js#L17) component to dispatch a custom event on `componentDidUpdate` and `componentDidMount`. 

We'd need to know when [this](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8efd9de68d277c1218eba52d289839c7e5786c1d/packages/documentation/src/layouts/module-components.js#L65) component changes but it looks like it's going to update whenever `mdxCodeBlock` changes which is observable at this layer. My understanding is that lifecycle calls bubble from the inside out i.e. `child.componentDidMount` is called before `parent.componentDidMount`.

Reply:
We can definitely look into this more.

Reply:
Yeah, I think we should see if we can import mermaid in layout.js and use componentDidMount in that component to do the initialization.